### PR TITLE
Admin setup page

### PR DESCRIPTION
**Pull request by gzinck**
Approve this PR after #37 is done (this builds on top of #37).

image

This adds the GUI for setting up an admin and organization. Animations have been created. I feel like I could run my own Pixar now with all this animation goodness.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,20 +1,42 @@
...
+    "@material-ui/icons": "^4.9.1",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.4.0",
+    "@testing-library/user-event": "^7.2.1",
+    "eslint-plugin-flowtype": "^3.13.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-scripts": "3.3.1",
+    "typescript": "^3.7.5"
```
Comment (evscott) [inline package.json]: Interesting dep! Are you going to be developing in typescript?

File: src/App.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+@keyframes App-logo-spin {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
```
Comment (evscott) [inline src/App.css]: I'm going to have to learn about some of these tags haha. I get the gist, though. Nice work!

File: package.json (hunk 1 above)
Comment (gzinck) [reply package.json]: I found that strange too! It's a dependency for some material UI elements—when I was installing material, it requested that I install typescript. I haven't actually needed it yet (as far as I am aware). I'll look into whether or not I can remove this dependency for a future PR!

File: src/App.css (hunk 2 above)
Comment (gzinck) [reply src/App.css]: Ha! This was all boilerplate code from Create React App—this is a good reminder to take out the CSS they put there!

File: package.json (hunk 1 above)
Comment (gzinck) [reply package.json]: Looks like that was my bad—I misread one of the errors when installing packages. I took out that dependency, should be good now!

File: src/App.css (hunk 2 above)
Comment (gzinck) [reply src/App.css]: Removed the boilerplate code in the new version!

File: package.json [hunk 3]
```diff
@@ -12,15 +14,23 @@
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "react-scripts": "3.3.1",
-    "typescript": "^3.7.5"
+    "react-scripts": "3.3.1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/evscott/hackapp-client.git"
```
Comment (gzinck) [inline package.json]: I also noticed that create react app removed some of the typical package.json items, so I've put them back in!

**Pull request merged**